tests/test_config.py

```diff
@@ -12,10 +12,9 @@
 from pykube import exceptions
 
 
-GOOD_CONFIG_FILE_PATH = os.path.sep.join(["tests", "test_config.yaml"])
-DEFAULTUSER_CONFIG_FILE_PATH = os.path.sep.join(
-    ["tests", "test_config_default_user.yaml"]
-)
+BASEDIR = Path("tests")
+GOOD_CONFIG_FILE_PATH = BASEDIR.joinpath("test_config.yaml")
```

Comment:
you can use the short form with the `/` operator:

```python
BASEDIR / "test_config.yaml"
```

Reply:
slick!